### PR TITLE
Update UG for adding high importance flag and listing contacts with high importance

### PR DESCRIPTION
**Pull request by xSaints19x**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -140,21 +140,24 @@ Examples:
 
 - `fav 1` — Adds contact at index 1 to your list of favourites
 
-### Adding high importance flag: `impt`
+### Adding high importance flag : `impt`
```
Comment (glennljw) [inline docs/UserGuide.md:143]: Great job keeping the format the same!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -140,21 +140,24 @@ Examples:
...
 
-Adds the contact to your list of people with high importance.
+Adds the contact to your list of people with high importance and a red flag will appear beside the contact's name to indicate that.
```
Comment (glennljw) [inline docs/UserGuide.md:145]: Might want to be consistent by using `contacts` instead of `people`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -140,21 +140,24 @@ Examples:
...
 
-Format:
-
-- `impt INDEX`
+Format: `impt INDEX`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-You can run `impt INDEX` where `INDEX` is the index of a contact that currently belongs in your high importance list to remove them.
+When a red flag appears beside the contact's name, you can run `impt INDEX` again where `INDEX` is the index of a contact that currently belongs in your list of people with high importance to remove them.
```
Comment (glennljw) [inline docs/UserGuide.md:150]: Might want to be consistent by using `contacts` instead of `people`

**Review by glennljw — CHANGES_REQUESTED** [review]
LGTM, just a few nits to fix! Great job overall 😄

File: docs/UserGuide.md (hunk 2 above)
Comment (xSaints19x) [reply docs/UserGuide.md]: Thanks for the reminder, fixed the required changes in the latest commit! 👍

File: docs/UserGuide.md (hunk 3 above)
Comment (xSaints19x) [reply docs/UserGuide.md]: Thanks for the reminder, fixed the required changes in the latest commit! 👍

**Review by glennljw — APPROVED** [review]

**Pull request merged**